src/common/SyncStatus.tsx

```diff
@@ -14,19 +14,29 @@ interface SyncStatusProps {
   syncStatus: SynchronizationStatus
 }
 
+const getSyncBlocksProgress = (syncStatus: SynchronizationStatusOnline): number => {
+  return (syncStatus.currentBlock / syncStatus.highestKnownBlock) * 100
```

Comment:
`highestKnownBlock` is always greater than 0?

Reply:
Will fix :)